README.md

```diff
@@ -29,14 +29,29 @@ Demonstrate your understanding of this week's concepts by answering the followin
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager
 
 1. Describe the biggest difference between `.forEach` & `.map`.
+    - .forEach: provides the function for each array element.
+    
+    - .map: creates a new array and provides the function on every element in the called array.
 
 2. What is the difference between a function and a method?
+    - Function: fundamental building block that preforms a task or calculates a value.
+
+    - Method: actions that can be performed on an object.
 
 3. What is closure?
+    - combination of functions within a lexical enviroment.
```

Comment:
More specifically, a closure is a nested function that has access to it's outter function's variables.  Closures store references to the outter functions variables even after function invocation has completed.